planning/behavior_path_planner/README.md

```diff
@@ -195,54 +196,153 @@ The Pull Over module is activated when goal is in the shoulder lane. Ego-vehicle
 
 #### **finish pull over condition** (need to meet any of the conditions below)
 
 - The distance to the goal from your vehicle is lower than threshold (default: < `1m`)
-- The speed of the vehicle is 0.
+- The ego-vehicle is stopped.
+  - The speed is lower than threshold (default: < `0.01m/s`).
 
-#### **Collision prediction with obstacles**
+#### General parameters for pull_over
 
-1. Predict each position of the ego-vehicle and other vehicle on the target lane of the pull over at t1, t2,...tn
-2. If a distance between the ego-vehicle and other one is lower than the threshold (`ego_velocity * stop_time (2s)`) at each time, that is judged as a collision
+| Name                       | Unit   | Type   | Description                                                                                                                             | Default value |
+| :------------------------- | :----- | :----- | :-------------------------------------------------------------------------------------------------------------------------------------- | :------------ |
+| request_length             | [m]    | double | when the ego-vehicle approaches the goal by this distance, the module is activated.                                                     | 100.0         |
+| th_arrived_distance_m      | [m]    | double | distance threshold for arrival of path termination                                                                                      | 1.0           |
+| th_stopped_velocity_mps    | [m/s]  | double | velocity threshold for arrival of path termination                                                                                      | 0.01          |
+| th_stopped_time_sec        | [s]    | double | time threshold for arrival of path termination                                                                                          | 2.0           |
+| pull_over_velocity         | [m/s]  | double | decelerate to this speed by the goal search area                                                                                        | 2.0           |
+| pull_over_minimum_velocity | [m/s]  | double | speed of pull_over after stopping once. this prevents excessive acceleration.                                                           | 0.3           |
+| margin_from_boundary       | [m]    | double | distance margin from edge of the shoulder lane                                                                                          | 0.5           |
+| decide_path_distance       | [m]    | double | decide path if it approaches this distance relative to the parking position. after that, no path planning and goal search are performed | 10.0          |
+| maximum_deceleration       | [m/s2] | double | maximum deceleration. Maximum deceleration. it prevents sudden deceleration when a parking path cannot be found suddenly                | 0.5           |
```

Comment:
The description of maximum_deceleration seems to be strange.
Maybe, `Maximum deceleration` is inserted incorrectly

Reply:
thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/1578/commits/0e7de761312528ac65945ae0e8292ed67ff146a3